tests/core/test_memory.py

```diff
@@ -97,10 +97,7 @@ def test_invalid_weights_summmary():
         Trainer(weights_summary='temp')
 
 
-@pytest.mark.parametrize(['mode'], [
-    pytest.param(ModelSummary.MODE_FULL),
-    pytest.param(ModelSummary.MODE_TOP),
-])
+@pytest.mark.parametrize('mode', [ModelSummary.MODE_FULL, ModelSummary.MODE_TOP])
 def test_empty_model_summary_shapes(mode):
```

Comment:
Not sure if we need typing here, but no typing for mode?
```suggestion
def test_empty_model_summary_shapes(mode: ModelSummary):
```